aztec/src/main/kotlin/org/wordpress/aztec/source/SourceViewEditText.kt

```diff
@@ -84,12 +88,16 @@ class SourceViewEditText : android.support.v7.widget.AppCompatEditText, TextWatc
         super.onRestoreInstanceState(savedState.superState)
         val customState = savedState.state
         visibility = customState.getInt("visibility")
+        val retainedContent = InstanceStateUtils.readAndPurgeTempInstance<String>(RETAINED_CONTENT_KEY, "", savedState.state)
+        setText(retainedContent)
     }
 
     override fun onSaveInstanceState(): Parcelable {
+        val bundle = Bundle()
+        InstanceStateUtils.writeTempInstance(context, null, RETAINED_CONTENT_KEY, text.toString(), bundle)
+        setText("")
```

Comment:
Same as in `AztecText`, this causes loss of text.

Reply:
Ahh I see it. I had tested with rotating the device, and enabling to not keep the activity and didn't notice the issue. 
The underlying problem (`onRestoreInstanceState` that is not called) was not surfaced in the previous version of the code since the component was still retaining the content internally and saving the data in `superState`.

Reply:
This is still a problem. If you switch to HTML, minimize and reopen the app the code is gone.

Reply:
Ohh 🤦‍♂️  Forgot to patch the HTML editor.